iocage_lib/ioc_check.py

```diff
@@ -56,9 +57,9 @@ def __init__(
         self.__check_fd_mount__()
         self.__check_datasets__()
 
-        self.pool_root_dataset = Dataset(self.pool, cache=False)
+        self.pool_root_dataset = Dataset(self.pool, cache=reset_cache)
```

Comment:
I don't get it, `cache=True` means "use cached values", `reset_cache=True` means "do not use cached values", doesn't it?

Reply:
Exactly, so if user said reset cache, we do that when the class is initialised and we don't have to do it each time we create a dataset which basically means that use the cache now as it has been reset already.

Reply:
I see, thank you for explanation.